deploy/02_land/01_set_land_admin.ts

```diff
@@ -15,16 +15,18 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   }
   if (currentAdmin) {
     if (currentAdmin.toLowerCase() !== landAdmin.toLowerCase()) {
-      await execute(
-        'Land',
-        {from: currentAdmin, log: true},
-        'changeAdmin',
-        landAdmin
+      await catchUnknownSigner(
+        execute(
+          'Land',
+          {from: currentAdmin, log: true},
+          'changeAdmin',
+          landAdmin
+        )
       );
     }
   }
 };
 export default func;
 func.runAtTheEnd = true;
-func.tags = ['Land', 'Land_setup'];
+func.tags = ['Land', 'Land_setup', 'LandAdmin'];
```

Comment:
Why would we need `LandAdmin` in the tags ?

Reply:
All the other tags were shared with other deployment scripts, should I change the name to something else? `Land_Admin_Setup` maybe?

Reply:
Ok i get it, no it's fine